pkg/ddevapp/drupal.go

```diff
@@ -427,6 +427,7 @@ func drupalConfigOverrideAction(app *DdevApp) error {
 		app.PHPVersion = nodeps.PHP81
 		drupalVersion = 9
 	case nodeps.AppTypeDrupal10:
+		app.CorepackEnable = false
```

Comment:
I think it should be disabled for all Drupal types below 11, so it would be easier to add this line before `switch`.

Thank you for your contribution.

Reply:
Makes sense, moved the line